packages/core/src/shared/utilities/pollingSet.ts

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { globals } from '..'
+import globals from '../extensionGlobals'
```

Comment:
avoids circular dependency.

Reply:
We will probably want to add a lint warning.

But for context what is happening is that `..` implicitly imports the `index.ts` module instead of the intended one and that causes the circular dependency issue.

Reply:
Is there every a case where importing from `index.ts` is preferred? Or is this a pretty straight-forward "don't do this" pattern?

Reply:
PR for lint rule: https://github.com/aws/aws-toolkit-vscode/pull/6372